### ISSUE TITLE
Make running benchmarks more user-friendly

### ISSUE THREAD

**Issue by Avasil** [monix/monix#928]
Our benchmarks require passing all the parameters (number of warmups, forks etc) in the command but we could add defaults as annotations so it's easier to run so instead of:
```scala
jmh:run -i 10 -wi 10 -f 2 -t 1 monix.benchmarks.TaskShiftBenchmark
```
We could just:
```scala
jmh:run monix.benchmarks.TaskShiftBenchmark
```
to keep the defaults unless we want to override them.

**Comment by TapanVaishnav** [comment]
@Avasil I would like to pick this one up.

**Comment by Avasil** [comment]
Solved in #936
